### ISSUE TITLE
Kint 3.2.1: Layout broken when using Kint over AJAX

### ISSUE THREAD

**Issue by Tilogorn** [kint-php/kint#315]
I just updated from 3.0 to 3.2.1 (after issue https://github.com/kint-php/kint/issues/314 got resolved) and suddenly have weird layout difficulties when using **Kint via AJAX**.

Assume a similiar environment as in Issue 314, just calling the PHP-Output via AJAX:

api.php:
```php
<?php

require __DIR__ . '/vendor/autoload.php';
Kint\Renderer\RichRenderer::$folder = false;

$myarray = array(
    0 => array(
        'prop1' => 'val1',
        'prop2' => 'val2',
        'prop3' => 'val3',
    ),
    1 => array(
        'prop1' => 'val12',
        'prop2' => 'val22',
        'prop3' => 'val32',
    ),
);

d($myarray);

?>

```
index.html
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <script src="https://code.jquery.com/jquery-3.4.0.min.js"
        integrity="sha256-BJeo0qm959uMBGb65z40ejJYGSgR7REI4+CW1fNKwOg=" crossorigin="anonymous"></script>
    <title>Kint-MWE</title>
</head>
<body>
    <button id="mybtn">Click</button>
    <div id="output"></div>
</body>
<script>
    $(document).ready(function () {
        $('#mybtn').on('click', function() {
            $.ajax({
                'url': 'api.php',
                'success': function(data) {
                    $('#output').html(data);
                }
            })
        });
    });
</script>
</html>

```
**First click on button**: Everything looks and behaves normally
![Bildschirmfoto vom 2019-05-02 11-24-03](https://user-images.githubusercontent.com/7524854/57066600-e1dd0f80-6ccc-11e9-8c5b-a47475ed6b4c.png)

**Second click on button**: No styles, nothing clickable
![Bildschirmfoto vom 2019-05-02 11-24-18](https://user-images.githubusercontent.com/7524854/57066644-089b4600-6ccd-11e9-897b-d8e18643fedd.png)

Back in 3.0 this worked without any problems. Can't test 3.1/3.2 because of Issue 314.

**Comment by jnvsor** [comment]
Thanks for the report and repro, I'll take a look at this now

**Comment by jnvsor** [comment]
@Tilogorn this is a bit of an odd problem. Kint now removes duplicate script/style tags (because they're inefficient in the case of dumping repeatedly to a page) which would be fine, except in your code you're overwriting the originals too.

I'll put a check for whether the first ones still exist in the code and that should fix it, but in the meantime you can set `RichRenderer::$always_pre_render = true` to fix your specific use-case

**Comment by jnvsor** [comment]
Fixed in c332db9529fdce74ae6a347f97a1e7395e76ced8 pushed 3.2.2

**Comment by Tilogorn** [comment]
You are incredibly fast fixing your issues. Please keep up your good work and a big THANK YOU! :-)
